transform.js

```diff
@@ -43,11 +44,19 @@ function toSDP(json) {
     let sdp = SDPUtils.writeSessionBoilerplate(json.sessionId, json.sessionVersion);
     sdp += 'a=msid-semantic:WMS *\r\n';
     sdp += json.media.map((m) => {
-        return SDPUtils.writeRtpDescription(m.kind, m.rtpParameters) +
-            'a=' + (m.direction || 'sendrecv') + '\r\n' +
+        var str = '';
+        if (m.kind === 'application' && m.protocol === 'DTLS/SCTP') {
+            str += 'm=application 9 DTLS/SCTP 5000\r\n' +
+                'c=IN IP4 0.0.0.0\r\n' +
+                'a=sctpmap:5000 webrtc-datachannel 256\r\n';
```

Comment:
hardcoding this stuff (to mozillas limit which is lower) is quite ugly however the whole sdp for this is terribly outdated